purchase_requisition_analytic/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+To use this module, you need to:
+
+#. Install the module
+#. Automatically propagate the purchase agreement analytic account in all RFQ lines if the field requisition_id is set.
```

Comment:
Please provide better steps to let a user see the module in action (even if it is automatic).